package.json

```diff
@@ -202,7 +202,7 @@
     "react-router": "^4.2.0",
     "react-sizeme": "^2.4.4",
     "react-slick": "^0.23.2",
-    "react-spring": "^5.7.2",
+    "react-spring": "^8.0.23",
```

Comment:
We're not actually using this anywhere, so version bump is ok

Reply:
Do we want to keep it consistent with Palette so we're not using two versions on import https://github.com/artsy/palette/blob/12597a2ec97fc760e4902229d9156aa444a9a7ef/packages/palette/package.json#L124 ?

Reply:
@sepans has a ticket to update Palette to latest react spring, so I think its ok if we have two versions for now, since we'll be migrating to the latest.